src/System.Diagnostics.Metrics.Temp/System.Diagnostics.Metrics.Temp.csproj

```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard2.0;net452</TargetFrameworks>
```

Comment:
Is this planned to get published to NuGet?
Assuming it's not, might want to throw:

```
<PropertyGroup>
  <IsPackable>false</IsPackable>
</PropertyGroup>
```

Assuming `dotnet pack` will run on the whole project and pushed somewhat automatically to nuget.org

Reply:
Will do! (we publish to myget automatically, and dont want to publish this)